docs/source/code_examples/tutorials/end-to-end/preparing_data_for_labelling.py

```diff
@@ -0,0 +1,134 @@
+"""
+Preparing your data for labelling
+=================================
+
+This example shows you how to go from your raw data to a project with your label
+structure (ontology) defined, your dataset attached, and ready to be labelled.
+
+Imports
+--------------
+"""
+# sphinx_gallery_thumbnail_path = 'images/end-to-end-thumbs/product-image.svg'
+
+from pathlib import Path
+
+from encord import EncordUserClient
+from encord.project import Project
```

Comment:
You can do `from encord import Project`. We should probably advertise this. Same for `Dataset`